Functions.Tests/GeoTrack.Tests.ps1

```diff
@@ -0,0 +1,26 @@
+$here = $PSScriptRoot
+$sut = $PSCommandPath.Replace('.Tests', '')
+. $sut
+
+
+Describe "Get-GoogleSessionVariable" {
+    It "Gets a WebRequestSession with Google session cookies" {
+        $session = Get-GoogleSessionVariable intellitectpowershelltests@gmail.com pstestaccount
```

Comment:
Perhaps consider moving this function to a new "Google.ps1" file?  I don't have other examples at the moment.  Hmmm... perhaps all the Google related code should be a in a Google.ps1 file that the geo stuff references?  Not sure.  :)